specs/deneb/beacon-chain.md

```diff
@@ -155,10 +157,24 @@ class ExecutionPayloadHeader(Container):
     withdrawals_root: Root
     data_gas_used: uint64  # [New in Deneb:EIP4844]
     excess_data_gas: uint64  # [New in Deneb:EIP4844]
+    parent_beacon_block_root: Root  # [New in Deneb:EIP4788]
 ```
 
 ## Helper functions
 
+### Beacon state accessors
+
+#### `get_previous_block_root`
+
+```python
+def get_parent_block_root(state: BeaconState) -> Slot:
+    if state.slot == GENESIS_SLOT:
+        slot = GENESIS_SLOT
+    else:
+        slot = Slot(state.slot - 1)
+    return state.block_roots[slot % SLOTS_PER_HISTORICAL_ROOT]
```

Comment:
this accessor is one entry point for getting the correct previous block root no matter where we are in the pathway from block production to block validation, and also handles genesis so that we do not underflow the computation